documentation/modules/adding-source-provider.adoc

```diff
@@ -19,8 +19,7 @@ You can add {a-rhv} source provider by using the {project-short} web console.
 
 .Prerequisites
 
-* CA certificate of the {manager}.
-* Apache CA certificate of the {manager}, in case the {manager} Apache CA certificate was replaced with a third-party CA certificate.
+* CA certificate of the {manager} or Apache CA certificate of the {manager} if the {manager} Apache CA certificate was replaced by a third-party CA certificate
```

Comment:
@RichardHoch 
For consistency throughout the doc:
"* {manager} CA certificate or {manager} Apache CA certificate if the {manager} CA certificate was replaced by a third-party certificate on the Apache server"